src/_utils/item/Item.tsx

```diff
@@ -47,40 +48,42 @@ export interface ItemProps {
   readonly hideHoverBackground?: boolean
 }
 
-const Item = ({
-  chevron,
-  className,
-  href,
-  onClick,
-  onBlur,
-  onFocus,
-  onMouseDown,
-  highlighted,
-  isClickable,
-  leftTitle,
-  leftTitleButtonAddon,
-  rightTitleStrikeThrough,
-  rightTitleAriaLabel,
-  leftTitleDisplay = TextDisplayType.TITLE,
-  leftTitleColor,
-  leftBody,
-  leftBodyDisplay = TextDisplayType.BODY,
-  leftBodyColor,
-  leftBodyAnnotation,
-  leftBodyAnnotationDisplay,
-  leftBodyAnnotationColor,
-  leftAddon,
-  rightTitle,
-  rightTitleDisplay = TextDisplayType.TITLE,
-  rightTitleColor,
-  rightBody,
-  rightBodyDisplay = TextDisplayType.BODY,
-  rightBodyColor,
-  rightAddon,
-  tag = <div />,
-  ariaLabel,
-  hideHoverBackground = false,
-}: ItemProps) => {
+const Item = (props: ItemProps) => {
+  const {
+    chevron,
+    className,
+    href,
+    onClick,
+    onBlur,
+    onFocus,
+    onMouseDown,
+    highlighted,
+    isClickable,
+    leftTitle,
+    leftTitleButtonAddon,
+    rightTitleStrikeThrough,
+    rightTitleAriaLabel,
+    leftTitleDisplay = TextDisplayType.TITLE,
+    leftTitleColor,
+    leftBody,
+    leftBodyDisplay = TextDisplayType.BODY,
+    leftBodyColor,
+    leftBodyAnnotation,
+    leftBodyAnnotationDisplay,
+    leftBodyAnnotationColor,
+    leftAddon,
+    rightTitle,
+    rightTitleDisplay = TextDisplayType.TITLE,
+    rightTitleColor,
+    rightBody,
+    rightBodyDisplay = TextDisplayType.BODY,
+    rightBodyColor,
+    rightAddon,
+    tag = <div />,
+    ariaLabel,
```

Comment:
This prop duplicates one in the A11yProps

Reply:
I'll do another ticket for this, I don't want to create breaking changes with this new component.